stdlib/2/os/__init__.pyi

```diff
@@ -10,7 +10,7 @@ from mypy_extensions import NoReturn
 error = OSError
 name = ...  # type: str
 
-class _Environ(MutableMapping[str, str]):
+class _Environ(MutableMapping[Union[str, unicode], Union[str, unicode]]):
```

Comment:
Reverted `os.environ` changes in e7bf317b3702ca7629d6003c68c63d92e3275455.  I opened the new issue in #872.

(My comment above is slightly incorrect, since I forgot to include `from __future__ import unicode_literals`; I corrected the example in the issue.)